code/mathematical-algorithms/prime_factors/prime_factors.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+int main()
+{
+	int num , isPrime;
```

Comment:
Add this at the beginning of your file, if you are assuming that the compiler this is going to be run on will not have bool:
```C
typedef int bool;
#define true 1
#define false 0
```

And then declare isPrime as a bool, and use true and false where needed.